src/Modules/ConfigOptions.lua

```diff
@@ -566,45 +566,76 @@ Huge sets the radius to 11.
+		if build.configTab.varControls['playerCursedWithProjectileWeakness'].shown then
+			modList:NewMod("ExtraCurse", "LIST", { skillId = "ProjectileWeakness", level = val, applyToPlayer = true, configCurse = true })
+		end
 	end },
-	{ var = "playerCursedWithPunishment", type = "count", label = "Punishment:", tooltip = "Sets the level of Punishment to apply to the player.", apply = function(val, modList, enemyModList)
-		modList:NewMod("ExtraCurse", "LIST", { skillId = "Punishment", level = val, applyToPlayer = true })
+	{ var = "playerCursedWithPunishment", type = "count", label = "Punishment:", tooltip = "Sets the level of Punishment to apply to the player.", saveShown = true, defaultPlaceholderState = 1, defaultHidden = true, removeBox = true, apply = function(val, modList, enemyModList, build)
+		if build.configTab.varControls['playerCursedWithPunishment'].shown then
+			modList:NewMod("ExtraCurse", "LIST", { skillId = "Punishment", level = val, applyToPlayer = true, configCurse = true })
+		end
 	end },
-	{ var = "playerCursedWithTemporalChains", type = "count", label = "Temporal Chains:", tooltip = "Sets the level of Temporal Chains to apply to the player.\nIn mid tier maps, 'of Temporal Chains' applies level 10.\nIn high tier maps, 'of Temporal Chains' applies level 15.", apply = function(val, modList, enemyModList)
-		modList:NewMod("ExtraCurse", "LIST", { skillId = "TemporalChains", level = val, applyToPlayer = true })
+	{ var = "playerCursedWithTemporalChains", type = "count", label = "Temporal Chains:", tooltip = "Sets the level of Temporal Chains to apply to the player.\nIn mid tier maps, 'of Temporal Chains' applies level 10.\nIn high tier maps, 'of Temporal Chains' applies level 15.", saveShown = true, defaultPlaceholderState = 10, defaultHidden = true, removeBox = true, apply = function(val, modList, enemyModList, build)
+		if build.configTab.varControls['playerCursedWithTemporalChains'].shown then
+			modList:NewMod("ExtraCurse", "LIST", { skillId = "TemporalChains", level = val, applyToPlayer = true, configCurse = true })
+		end
 	end },
-	{ var = "playerCursedWithVulnerability", type = "count", label = "Vulnerability:", tooltip = "Sets the level of Vulnerability to apply to the player.\nIn mid tier maps, 'of Vulnerability' applies level 10.\nIn high tier maps, 'of Vulnerability' applies level 15.", apply = function(val, modList, enemyModList)
-		modList:NewMod("ExtraCurse", "LIST", { skillId = "Vulnerability", level = val, applyToPlayer = true })
+	{ var = "playerCursedWithVulnerability", type = "count", label = "Vulnerability:", tooltip = "Sets the level of Vulnerability to apply to the player.\nIn mid tier maps, 'of Vulnerability' applies level 10.\nIn high tier maps, 'of Vulnerability' applies level 15.", saveShown = true, defaultPlaceholderState = 10, defaultHidden = true, removeBox = true, apply = function(val, modList, enemyModList, build)
```

Comment:
```suggestion
	{ var = "playerCursedWithVulnerability", type = "count", label = "Vulnerability:", tooltip = "Sets the level of Vulnerability to apply to the player.\nIn mid tier maps, 'of Vulnerability' applies level 10.\nIn high tier maps, 'of Vulnerability' applies level 15.", saveShown = true, defaultPlaceholderState = 10, defaultHidden = true, removeBox = true, apply = function(val, modList, enemyModList, build)
```
These tooltips are outdated see https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/5560